core/math/color.h

```diff
@@ -302,4 +302,7 @@ _FORCE_INLINE_ Color operator*(float p_scalar, const Color &p_color) {
 	return p_color * p_scalar;
 }
 
+template <>
+struct is_zero_constructible<Color> : std::true_type {};
+
```

Comment:
Is this applicable for `Color`? Its default constructor isn't all zeros, the alpha initializes as `1`.

Reply:
You're totally right, I missed that. Good catch!